api/cms.go

```diff
@@ -19,10 +19,15 @@ func CMSPerson(c *gin.Context) {
 	if c.Bind(&r) != nil {
 		return
 	}
-	id := c.Param("id")
+	id := c.Query("id")
```

Comment:
why not keep the REST approach ?
path param (`/:id`) instead query params

Reply:
done

Reply:
done